hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -106,6 +106,12 @@
   private final OzoneManager impl;
   private OzoneManagerDoubleBuffer ozoneManagerDoubleBuffer;
 
+
+  public OzoneManagerRequestHandler(OzoneManager om) {
```

Comment:
Should the isRatisEnabled flag passed here and then we should assign the ozoneManagerDoubleBuffer using a getter in the OzoneManagerProtocolServerSideTranslatorPB.java ?

Reply:
I think end result will be the same. If you want that way I can change it? Any advantage with that?

Reply:
@mukul1987 can you please answer @bharatviswa504's question?

Reply:
Sorry I missed this earlier. the suggestion was wrt code cleanliness, yes the results will still be the same.